index.js

```diff
@@ -63,7 +63,7 @@ module.exports = function (mcVersion, preNetty) {
   nmcData.isOlderThan = version => nmcData.version['<'](version)
   nmcData.version = Object.assign(majorVersion, nmcData.version)
   cache[majorVersion.type + '_' + majorVersion.majorVersion] = nmcData
-
+  nmcData.supportFeature = feature => require('./lib/supportsFeature')(feature, nmcData.version.minecraftVersion)
```

Comment:
let's do require('./lib/supportsFeature') at the top level

Reply:
done :)